mobile/cmp/navigator/NavigatorModel.ts

```diff
@@ -52,31 +57,40 @@ export class NavigatorModel extends HoistModel {
     stack: PageModel[] = [];
 
     pages: PageConfig[] = [];
-
     track: boolean;
-    swipeToGoBack: boolean;
     pullDownToRefresh: boolean;
+    transitionMs: number;
     renderMode: RenderMode;
     refreshMode: RefreshMode;
 
-    private _navigator = null;
+    private _swiper: Swiper;
     private _callback: () => void;
-    private _prevKeyStack: string[];
 
     get activePageId(): string {
         return this.activePage?.id;
     }
 
     get activePage(): PageModel {
-        const {stack} = this;
-        return stack[stack.length - 1];
+        return this.stack[this.activePageIdx];
+    }
+
+    get activePageIdx(): number {
```

Comment:
curious -- why default to end of stack instead of zero?

Reply:
The active page is always the last page in the stack, e.g. Home > Grids > Grid Detail

Reply:
And now that I've written this, I realise that before this._swiper is available the stack is empty so that's a moot point :) Will simplify to zero 👍